src/arweave/composite-client.ts

```diff
@@ -364,16 +364,16 @@ export class ArweaveCompositeClient
     }
   }
 
-  selectChunkPeers(peerCount: number): string[] {
-    const log = this.log.child({ method: 'selectChunkPeers' });
+  selectPeers(peerCount: number): string[] {
```

Comment:
We should use a separate weights for chunks and headers. I expect headers to be much more widely available than chunks. We don't want header availability affecting chunk weights.

Reply:
done, I solved this by passing a string matching the class property name of the weighter peers list